src/cpp/src/speculative_decoding/speculative_decoding_impl.cpp

```diff
@@ -47,11 +47,12 @@ ContinuousBatchingPipeline::SpeculativeDecodingImpl::SpeculativeDecodingImpl(con
     if (is_scheduler_undefined) {
         // split KV cache to 2 caches for main and draft models
         size_t main_model_cache_size = utils::get_kv_cache_size(main_model),
-            draft_model_cache_size = utils::get_kv_cache_size(draft_model);
+               draft_model_cache_size = utils::get_kv_cache_size(draft_model);
```

Comment:
get_kv_cache_size => get_hidden_size

it's not KV cache size, but size of hidden dimension